apps/meteor/ee/server/api/licenses.ts

```diff
@@ -71,7 +71,7 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'licenses.isEnterprise',
-	{ authRequired: true },
```

Comment:
Notice that even with this change in place, the `method.callAnon/license:isEnterprise` is being used 👀 is that expected?